src/callbacks_step/save_restart.jl

```diff
@@ -156,5 +160,32 @@ function load_restart_file(semi::AbstractSemidiscretization, restart_file)
     load_restart_file(mesh_equations_solver_cache(semi)..., restart_file)
 end
 
+"""
+    load_controller!(integrator, restart_file::AbstractString)
+
+Load the context information for PID- and PIControllers saved in a `restart_file`.
```

Comment:
```suggestion
Load the context information for error-based time integrators saved in a `restart_file`.
```
Less technical and more informative to users who don't know any details